File: Salary.java
import java.util.Scanner;
class Salary{
public static void main(String args[]){
Scanner cin = new Scanner(System.in);
System.out.println("Welcome");
System.out.println("How can I help you");
System.out.println("Please Entre Your Salary and let me check your Designation");

int pay;
pay = cin.nextInt();
System.out.println("Pay: " + pay);
if (pay>=10000&&pay<=20000){
System.out.println("Manager Operations");}

if (pay>=20000&&pay<=30000){
System.out.println("Manager");}

if (pay>=30000&&pay<=40000){
System.out.println("Area Manager");}

if (pay>=40000&&pay<=50000){
System.out.println("Regional Manager");}

else{
    System.out.println("Invalid!");
}
}
}

File: C++/Salary.cpp
#include<iostream>
using namespace std;
int main(){
    cout<<"Welcome"<<endl;
cout<<"How can I help you"<<endl;
cout<<"Please Entre Your Salary and let me check your Designation"<<endl;

int pay;
cin>>pay;
cout<<"Pay: "<<pay<<endl;
if (pay>=10000&&pay<=20000){
cout<<"Manager Operations"<<endl;}

if (pay>=20000&&pay<=30000){
cout<<"Manager"<<endl;}

if (pay>=30000&&pay<=40000){
cout<<"Area Manager"<<endl;}

if (pay>=40000&&pay<=50000){
cout<<"Regional Manager"<<endl;}

else {
    cout<<"Invalid!"<<endl;
}
}
}